mesonbuild/compilers/c.py

```diff
@@ -168,21 +168,21 @@ def get_include_args(self, path, is_system):
     def get_std_shared_lib_link_args(self):
         return ['-shared']
 
-    def get_library_dirs_real(self):
-        env = os.environ.copy()
-        env['LC_ALL'] = 'C'
-        stdo = Popen_safe(self.exelist + ['--print-search-dirs'], env=env)[1]
-        paths = []
-        for line in stdo.split('\n'):
-            if line.startswith('libraries:'):
-                libstr = line.split('=', 1)[1]
-                paths = [os.path.realpath(p) for p in libstr.split(':') if os.path.exists(os.path.realpath(p))]
-        return paths
-
-    def get_library_dirs(self):
-        key = tuple(self.exelist)
+    def get_library_dirs_real(self, env):
+        code=''
```

Comment:
__[Flake8]__

[E225] missing whitespace around operator


<!-- Comment by [Sider](https://sider.review) -->

Reply:
Sure. I didn't run flake8 and clean up the code much, as I'm looking for initial feedback as to whether this a correct approach or not to fixing the issues.  [Hence "DRAFT" in the title :-)]